person.rb

```diff
@@ -2,17 +2,18 @@
 require_relative 'decorator'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ]  rentals should have both setters and Getters.  Use attr_accessor to provide those methods than using attr_reader which provide getters method only.